src/baseline_bots/bots/smart_order_accepter_bot.py

```diff
@@ -101,6 +119,9 @@ def __call__(self, rcvd_messages: List[Tuple[int, Message]]):
         orders_data = OrdersData()
         orders_data.add_orders(best_orders)
 
-        # generate messages
+        # generate messages for FCT sharing info orders
         messages = self.gen_messages(orders_data)
+
+        # generate proposal response YES/NO to allies
+        messages = self.gen_proposal_reply(best_proposer, prp_orders, messages)
```

Comment:
since gen_proposal_reply doesnt seem to return anything, doesnt messages get set to None?

Reply:
I forgot to return MessageData. thanks for reviewing!